.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Let's not commit this file.

Reply:
Unless this is intentional?

Reply:
No it is committed by mistake, I'll remove it now